submissions/krylenger/html-css-popup/style.css

```diff
@@ -0,0 +1,244 @@
+* {
+    font-family: monospace, Helvetica, sans-serif, Arial;
+    font-size: 17px;
+    font-weight: 500;
+    margin: 0px;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+}
+
+a {
+    text-decoration: none;
+    outline: none;
+}
+
+img {
+    width: 75%;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+span {
+    text-align: center;
+}
+
+label {
+    width: 100%;
+}
+
+label:hover {
+    cursor: pointer;
+}
+
+.navigation-sub-menu .checkbox {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  background-color: #eee;
+  opacity: 0;
+}
+
+.checkbox:hover {
+    cursor: pointer;
+}
+
+.navigation-list-icons {
+    background-size: 35px 35px;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 100%;
+}
+
+.toggle-pop-up-label {
+    background-image: url(./icons/menu.png);
+}
+
+.navigation-notification {
+    background-image: url(./icons/notification.png);
+}
+
+.navigation-profile {
+    background-image: url(./icons/profile.jpg);
+}
+
+.toggle-pop-up:focus + label {
+    outline: 2px solid rgb(77, 77, 77);
+}
+
+.navigation-list > li > a:focus {
+    outline: 2px solid rgb(77, 77, 77);
+}
+
+.navigation-container { 
+    display: flex;
+    width: 100%;
+    height: 50px;
+    flex-direction: row;
+    justify-content: flex-end;
+}
+
+.navigation-list-container {
+    width: 320px;
+    height: 100%;
+}
+
+.pop-up-container {
+    display: none;
+    text-align: center;
+    height: 420px;
+    width: 270px;
+    border: solid 1px lightgrey;
+    border-radius: 5px;
+    box-shadow: 0px 0px 5px 0px rgba(0,0,0,1);
+}
+
+.pop-up-icons-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-content: center;
+    height: 94%;
+}
+
+ul {
+    display: flex;
+    justify-content: space-around;
+    margin: 0;
+    padding-inline-start: 0;
+}
```

Comment:
Avoid the name of the elements, it's better to use the class name.

CSS validation has an error near property `padding-inline-start`, you can see follow the [link](https://jigsaw.w3.org/css-validator/validator?uri=https%3A%2F%2Fkrylenger.github.io%2Ffrontend-2021-homeworks%2Fsubmissions%2Fkrylenger%2Fhtml-css-popup%2F&profile=css3svg&usermedium=all&warning=1&vextwarning=&lang=ru)

Reply:
✅ done